packages/DataGrid/src/DataGrid.js

```diff
@@ -10,11 +10,16 @@ import * as sc from "./DataGrid.styles";
 import Basement, { End } from "./components/Basement";
 import InfiniteScroll from "./components/InfiniteScroll";
 import { Row, HeaderRow, StickyRow, StickyHeaderRow, createItemData } from "./DataGrid.Rows";
+import getScrollbarWidth from "./helpers/getScrollbarWidth";
 
 const propTypes = {
   /** If the data cell should automatically get focus  */
   autofocus: PropTypes.bool,
+  /** Define the look for borders in the table */
+  borderType: PropTypes.oneOf(["grid", "empty", "horizontal", "vertical"]),
   children: PropTypes.node.isRequired,
+  /** This will force the table to include in the calculation of the table the scrollbar thickness */
+  forceTableWidthWithScrollBars: PropTypes.bool,
```

Comment:
Is this like `widthIncludesScrollBar`?

Reply:
no really, look a the images on the description of the PR, this is because if you have a dynamic table that can have scrollbar dynamically you should have to take into account that scrollbar, to avoid an undesired overflow.